.circleci/config.yml

```diff
@@ -58,13 +58,13 @@ commands:
     description: Run `kedro run` end to end tests for all starters
     steps:
       - run:
-          command: behave features/run.feature
+          command: behave features/run.feature --summary
```

Comment:
Is this part of the starters changes, or is there some other motivation?

Reply:
I added this initially because the e2e test log is too noisy, this doesn't seem to help so I will revert this